share/man/man4/ixgbe.4

```diff
@@ -124,3 +124,11 @@ The
 .Nm
 driver was written by
 .An Intel Corporation Aq Mt freebsd@intel.com .
+.Sh CAVEATS
+Intel (R) Flow director support is not fully implemented in
+.Fx
+at this time and additional work is required
+before those features can be supported.
+.Pp
+Enabling flow director may route traffic to the wrong RX queue of the NIC,
```

Comment:
@concussious Sorry I missed one other critique, we don't have any support for fdir in the driver.  The OS-independent HAL support code exists but is not wired up at all as far as I can tell  so this entire sentence can be dropped.

Reply:
Very cool, thank you!